lib/widgets/atoms/navigation_bar.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+
+class CustomizedBottomNavigationBar extends StatefulWidget {
+  const CustomizedBottomNavigationBar({Key? key, required this.child})
```

Comment:
We should write constructor in a way that uses `super.key`. This can be replaced with: `const CustomizedBottomNavigationBar({super.key, required this.child});`

Reply:
Done!